product-service/environment.ts

```diff
@@ -0,0 +1,7 @@
+export const environment = {
```

Comment:
Please don't checkin env related files to git. Move then to .env and place in .ignore